server/src/database/index.js

```diff
@@ -11,7 +11,7 @@ const connect = async () => {
   while (1) {
     try {
       logger.debug('Trying to connect to database');
-      await mongoose.connect('mongodb://mongo:27017/your_spotify', { useNewUrlParser: true });
+      await mongoose.connect(process.env.MONGO_ENDPOINT, { useNewUrlParser: true });
```

Comment:
Maybe write `process.env.MONGO_ENDPOINT || 'mongo'` here so that the user is not compelled to define it

Reply:
Since the environment variable has a default value this would be dead code.

Reply:
Ok !

Reply:
Ok, haven’t thought about that

```suggestion
      const fallbackConnection = 'mongodb://mongo:27017/your_spotify'
      await mongoose.connect(process.env.MONGO_ENDPOINT || fallbackConnection, { useNewUrlParser: true });
```